### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Reflow: Two scroll bar appearing for whole page at 400% zoom

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/vscode-docs#5129]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Reflow;#WCAG1.4.10; #FTP; #ChromiumEdge;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Pre-requisites:**
Set the Display Resolution : 1280*1024.

**Repro Steps:**

1. Launch the application Visual Studio Code Service.
2. Observe that two scroll bar appearing for whole page at 400% zoom.

**Actual:**
Two scroll bar appearing for whole page at 400% zoom.
**NOTE:**
Same issue is observed for all the pages.

**Expected Result:**
There should be only one scroll bar for whole page at 400% zoom.

**User Impact:**
Low vision users who zoom the page for better visibility will find it difficult to view the content if two dimensional scroll bars are provided.

**Attachment:**
![A11y_Visual Studio Code Services_Home_Reflow (2)](https://user-images.githubusercontent.com/97668775/151743937-05c4ee94-b85d-4fbb-8ac6-e54384941f35.png)

**Comment by gregvanl** [comment]
Duplicate of issues https://github.com/microsoft/vscode-docs/issues/5102 and https://github.com/microsoft/vscode-docs/issues/3432

**Comment by rzhao271** [comment]
I've some ideas for this one.

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
Please find below attachment for reference.
Github tag: #Closed;
![5129_Fixed](https://user-images.githubusercontent.com/97668775/218079079-7cefbede-df2f-4db6-a50d-797e93666858.gif)
